app/services/panoptes_api.rb

```diff
@@ -30,14 +37,25 @@ def project(slug)
     { id: response['id'].to_i, slug: response['slug'] }
   end
 
+  def workflow(id, include_project=nil)
+    query = { id: id }
+    response = if include_project
+      query[:include] = 'project'
+      get_workflow_with_project(query)
+    else
+      get_workflow(query)
```

Comment:
Is this used right now, will it be in the future?

Reply:
yes, see above ^^^